docs/ext/commands/commands.rst

```diff
@@ -584,7 +584,7 @@ When mixed with the :data:`typing.Optional` converter you can provide simple and
                        reason: str):
         """Mass bans members with an optional delete_days parameter"""
         for member in members:
-            await member.ban(delete_message_days=delete_days, reason=reason)
+            await member.ban(clean_history_duration=delete_days * 86400, reason=reason)
```

Comment:
this makes more sense: 
```
timedelta(days=flags.days)
```

(needs to import timedelta)

Reply:
https://github.com/DisnakeDev/disnake/pull/659/commits/f2a7ffb88cb8ed1e0253602e69be1aad267ac132